### ISSUE TITLE
Write text on images as an augmentation

### ISSUE THREAD

**Issue by hyperfraise** [pytorch/pytorch#75513]
### 🚀 The feature, motivation and pitch

Hi. 

Right now, I believe that if you wanna write text on a GPU Tensor, you're gonna have to convert it back to numpy array and use OpenCV.

This is unfortunate since text writing is a very good augmentation is some cases, where input data might have timestamps for example. Also, most efficient loading libraries make use of the GPU (for example Nvidia Dali, decord) for efficient loading, meaning that converting a tensor back to numpy array sacrifices this advantage they have for "large" training (when you don't have the whole dataset in RAM).

I think it'd be great if writing random text on an image was a torchvision feature :D

### Alternatives

Convert the tensor it back to numpy array and use OpenCV.
